docker-compose.yml

```diff
@@ -0,0 +1,179 @@
+    ports:
+      - "127.0.0.1:8554:8554"
+  camera-simulator0:
+    build:
+      context: .
+    image: openvino/ubuntu20_data_runtime:2021.4.2
+    container_name: camera-simulator0
+    network_mode: "host"
+    entrypoint: ffmpeg
+    command: "
+        -nostdin
+        -re -stream_loop -1
+        -i /home/pipeline-server/sample-media/coca-cola-4465029-3840-15-bench.mp4
+        -c copy
+        -f rtsp
+        -rtsp_transport
+        tcp
+        rtsp://localhost:8554/camera_0
+        "
+    depends_on:
+      - camera-simulator
+    volumes:
+      - ./sample-media:/home/pipeline-server/sample-media
+  camera-simulator1:
+    build:
+      context: .
+    image: openvino/ubuntu20_data_runtime:2021.4.2
+    container_name: camera-simulator1
+    network_mode: "host"
+    entrypoint: ffmpeg
+    command: "
+        -nostdin
+        -re -stream_loop -1
+        -i /home/pipeline-server/sample-media/couple-paying-at-the-counter-in-the-grocery-4121754-3840-15-bench.mp4
+        -c copy
+        -f rtsp
+        -rtsp_transport
+        tcp
+        rtsp://localhost:8554/camera_1
+        "
+    depends_on:
+      - camera-simulator
+    volumes:
+      - ./sample-media:/home/pipeline-server/sample-media
+  camera-simulator2:
+    build:
+      context: .
+    image: openvino/ubuntu20_data_runtime:2021.4.2
+    container_name: camera-simulator2
+    network_mode: "host"
+    entrypoint: ffmpeg
+    command: "
+        -nostdin
+        -re -stream_loop -1
+        -i /home/pipeline-server/sample-media/vehicle-bike-3840-15-bench.mp4
+        -c copy
+        -f rtsp
+        -rtsp_transport
+        tcp
+        rtsp://localhost:8554/camera_2
+        "
+    depends_on:
+      - camera-simulator
+    volumes:
+      - ./sample-media:/home/pipeline-server/sample-media
+  OvmsServer:
+    image: openvino/model_server:2023.1-gpu
+    container_name: ovms-server0
+    restart: always
+    ports:
+      - '9001:9001'
+      - '9002:9002'
+    command: 
+      - '--config_path'
+      - '/models/config.json'
+      - '--port'
+      - '9001'              
+      - '--rest_port'
+      - '9002'
+      - '--log_level'
+      - 'INFO'
+    volumes:
+      - ./configs/opencv-ovms/models/2022:/models
+    environment:
+      - DEVICE=CPU update_config:dev
+      - PLATFORM="core"
+      - server_cl_cache_dir=/home/pipeline-server/.cl-cache   
+  OvmsClient0:
+    image: python-demo:dev
+    container_name: object-detection0
+    network_mode: "host"
+    entrypoint: /opencv-ovms/demos/object_detection/python/entrypoint.sh
+    environment:
+      - CONTAINER_NAME="object-detection0"
+      - GRPC_PORT=9001
+      - INPUTSRC=rtsp://localhost:8554/camera_0
+      - cid_count=0
+      - DETECTION_MODEL_NAME=ssd_mobilenet_v1_coco
+      - DETECTION_LABEL_FILE=coco_91cl_bkgr.txt
+      - DETECTION_ARCHITECTURE_TYPE=ssd
+      - DETECTION_OUTPUT_RESOLUTION=1280x720
+      - DETECTION_THRESHOLD=0.50
+      - RENDER_MODE=0 #RENDER_MODE=1 doesn't work
```

Comment:
I tested this and it works. You need to run `xhost +local:docker` first and add this env variable: - DISPLAY=$DISPLAY

Reply:
Good to know, will update the comments. However developer toolbox won't support it.

Reply:
ok, at least add the DISPLAY=$DISPLAY and remove the comment that RENDER_MODE=1 doesn't work please.